mmf/models/vilbert.py

```diff
@@ -835,19 +918,38 @@ def __init__(self, config):
 
         self.init_weights()
 
+    @torch.jit.unused
+    def _change_dtype_for_fp16(self, extended_attention_mask):
+        """
+        change attention tensor's precision for fp16 training only
+        """
+        if self.training:
+            extended_attention_mask = extended_attention_mask.to(
+                dtype=next(self.parameters()).dtype
+            )  # fp16 compatibility
+        return extended_attention_mask
```

Comment:
This is not required with latest pytorch 1.6+. We can guard blocks using 'torch.jit.is_scripting()'. Check the implementation in VisualBERT model.